### DISCUSSION TITLE
Value Relation Widget doesn't work anymore

### DISCUSSION THREAD

**Discussion by aliasatp** [opengisch/QField#5008] [Bugs]
Hi,
today after new sync my fields set as Value Relation Widget doen't work as they should. They appear as no editalble text with no multi-selection options and relate list options. It afflicts all my projects and it starter today with new sync.

**Comment by nirvn** [comment]
@aliasatp , can you share a project file?

**Reply by aliasatp** [reply]
I realised it doesn't pack no geometry table anymore (the tables are empty in qfield) so value relation widget doesn't have related table! no changes I have done, no "today" syncs projects work as they should.

**Reply by aliasatp** [reply]
I confirm. No geometry tables are packed as "empty"! I have different projects but the issue is in all of them

**Reply by nirvn** [reply]
@aliasatp , by 'syncs' you mean synchronizing cloud project right?

**Reply by aliasatp** [reply]
yes, first button. I cofirm my no geomerty tables are emtpies. The projects in qfield after "synchronizing cloud project" have all the tables as it was just untill few days ago but they are empties so everything related with them doesn't work as it should.
If I push back data the tables are populated

**Reply by aliasatp** [reply]
Geometry tables are packed correctly

**Reply by aliasatp** [reply]
I re-create a project but the issue remains. No geometry tables are always empties. Have you got any ideas? We are in productions.

**Comment by ThomasDV1984** [comment]
I have the same issue, relationships not longer working due to empty no geometry tables.

**Comment by BoswachterMarc** [comment]
See also comment of @guibeatle https://github.com/opengisch/qfieldcloud/issues/874#issuecomment-1928083350

**Comment by suricactus** [comment]
Hey! The problematic piece of code has been detected. Fix is being implemented. Will keep you updated.

**Comment by suricactus** [comment]
Fix is on the way, should be deployed by the end of the day.

Will text you in https://github.com/opengisch/qfieldcloud/issues/874#issuecomment-1928083350 when it is deployed. If you have any follow-ups, please use the issue linked on QFieldCloud repo.

Thanks for reporting, finding a workaround and appologies for the inconvenience.
